README.md

```diff
@@ -29,7 +29,7 @@ pip install -U tensorflow-cloud
 ```shell
 git clone https://github.com/tensorflow/cloud.git
 cd cloud
-pip install .
+pip install tensorflow_cloud/python/.
 ```
 
```

Comment:
Do you need to keep `tensorflow_cloud` directory above `python`?

IIUC, you needed `tensorflow_cloud` as the root Python package directory. Since Python package root is now `./python/tensorflow_cloud`, I think you can dispense it.

A s we discussed offline, if we were to build the wheel by Bazel, it makes sense to keep the top-level `tensorflow_cloud`, something like https://github.com/tensorflow/ranking/tree/master/tensorflow_ranking, but it will be a major change in build tooling so I don't propose us to prepare for it until we decide to do so.

Reply:
I felt it is cleaner to keep all the main code/logic of the repo inside one root directory `tensorflow_cloud`.

Reply:
what other body of code do you envision to belong to directories under `tensorflow/cloud(root)/...` but not under `tensorflow/cloud(root)/tensorflow_cloud`?

also, do you envision that a body of code that aren't distributed as the part of `tensorflow_cloud` Python (PIP) package, but constitute own build unit (via Bazel, etc), would also belong to `tensorflow/cloud(root)/tensorflow_cloud/...` ? 


(bottom line; I think it doesn't hurt to keep one more layer, but I'm simply curious what would be at the same level as `tensorflow/cloud/tensorflow_cloud`.

Reply:
At this time it will be most of the files which are already outside, for example github workflows, contributing guidelines, readme, license, images etc. ie. Files which wouldn't belong to any distribution.

Reply:
If I'm following correctly does that mean ```tensorflow/cloud/tensorflow_cloud``` will always only have ```python``` in it. All read me files etc will be at ```tensorflow/cloud/ ```. 

If this is a correct understanding would it make sense to collapse ```tensorflow/cloud/tensorflow_cloud/python/tensorflow_cloud``` to just ```tensorflow/cloud/python/tensorflow_cloud```?

Reply:
> If I'm following correctly does that mean `tensorflow/cloud/tensorflow_cloud` will always only have `python` in it. All read me files etc will be at `tensorflow/cloud/ `.

Root folder: https://github.com/tensorflow/cloud/tree/master/tensorflow_cloud
Python projects: https://github.com/tensorflow/cloud/tree/master/tensorflow_cloud/python/tensorflow_cloud
Keras for GCP: https://github.com/tensorflow/cloud/tree/master/tensorflow_cloud/python/tensorflow_cloud/core/...
Cloud Tuner: https://github.com/tensorflow/cloud/tree/master/tensorflow_cloud/python/tensorflow_cloud/cloud_tuner/...
Cpp: https://github.com/tensorflow/cloud/tree/master/tensorflow_cloud/cpp
Monitoring ((as an example for non-python project)): https://github.com/tensorflow/cloud/tree/master/tensorflow_cloud/cpp/monitoring/...

Images, Github workflows, Main repo README, CONTRIBUTING guidelines etc will continue to live under root folder.

Reply:
I'm cool with the proposed structure, since ./cpp in its entirety isn't part of the `tensorflow_cloud` distribution, I didn't find it is strictly necessary. Still a good argument that source tree of the library, regardless of python or cpp, to be isolated in one sub-directory. Other good name could have been 'src', etc.

Reply:
Thank you for detailed description... lgtm as well!

Reply:
Renamed to src as per Makoto's other comment.